src/app/pages/landing-page/landing-page.component.html

```diff
@@ -6,7 +6,7 @@ <h1>The future of irrigation is now</h1>
     <a href="" class="secondary">Learn More</a>
   </app-hero>
 
-  <app-frame [vm]="vm.frameVMs[0]">
+  <app-frame id="products" [vm]="vm.frameVMs[0]">
```

Comment:
Doesn't the frameVM has an id property already? If not make one. Do not hardcode this id here as it disconnects some logic and makes you type the product id twice which can lead to typos/errors.